.github/workflows/ci.yml

```diff
@@ -39,20 +39,57 @@ jobs:
         version: ["3.1.4"]
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@ec02537da5712d66d4d50a0f33b7eb52773b5ed1
-        with:
-          ruby-version: '3.1'
-      - name: Install dependencies
-        run: bundle install
+        uses: actions/checkout@v4
+      - name: Output CHANGELOG
+        run: bin/print_changelog "${{matrix.version}}"
+      - name: Build Docker image
+        run: bin/activate_docker "${{matrix.stack}}"
+      - name: Build and package Ruby runtime
+        run: bin/build_ruby "${{matrix.stack}}" "${{matrix.version}}"
+      - name: Verify ruby executable and output rubygems version
+        run: bin/print_gem_version "${{matrix.stack}}" "${{matrix.version}}"
+
+  integration_test-heroku-24-amd:
+    runs-on: pub-hk-ubuntu-22.04-xlarge
+    strategy:
+      matrix:
+        stack: ["heroku-24"]
+        version: ["3.2.3"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Output CHANGELOG
+        run: bin/print_changelog "${{matrix.version}}"
```

Comment:
@schneems You should have a look at what @dzuelke does with [Job Summaries](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/) the PHP automation. It would save you from having to open up the individual step output to get at this content.

Reply:
That's neat! I implemented it, but the information didn't make it's way to the summary and ended up being hidden from that job's output as well. https://github.com/heroku/docker-heroku-ruby-builder/actions/runs/8527165876/job/23357986389?pr=38 I'll port that over to a new PR to play with instead of blocking this one